docs/glossary/t/transfer-mode.lcdoc

```diff
@@ -12,6 +12,35 @@ Some transfer modes make the <object(glossary)> partly or completely
 transparent by combining each <pixel|pixel's> color with the color
 underneath it.
 
+The following is a non-exhaustive list of transfer modes:
+
+* <addMax(keyword)>
+* <adMin(keyword)>
+* <addPin(keyword)>
+* <addOver(keyword)>
+* <blend(keyword)>
+* <clear(keyword)>
+* <noOp(keyword)>
+* <notSrcAnd(keyword)>
+* <notSrcAndReverse(keyword)>
+* <notSrcBic(keyword)>
+* <notSrcCopy(keyword)>
+* <notSrcOr(keyword)>
+* <notSrcOrReverse(keyword)>
+* <notSrcXOr(keyword)>
+* <reverse(keyword)>
+* <set(keyword)>
+* <srcAnd(keyword)>
+* <srcAndReverse(keyword)>
+* <srcBic(keyword)>
+* <srcCopy(keyword)>
+* <srcOr(keyword)>
+* <srcOrReverse(keyword)>
+* <srcXOr(keyword)>
+* <subOver(keyword)>
+* <subPin(keyword)>
+* <transparent(keyword)>
+
 References: object (glossary), pixel (glossary)
```

Comment:
Done this, ready to go. Thanks for all your guidance. My head is finally getting around all of this. I must be getting old.